src/Workspaces/Remote/Core/ProjectSystem/IWorkspaceProjectSystemService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Remote.ProjectSystem;
+
+internal interface IWorkspaceProjectSystemService
```

Comment:
I wonder if ILanguageServerProjectSystemService might be a more reasonable name. Or may be just IProjectSystemService?

Reply:
Ah yeah, since this is language server only.